packages/ember-htmlbars/tests/helpers/if_unless_test.js

```diff
@@ -316,71 +70,6 @@ QUnit.test('properties within an if statement should not fail on re-render', fun
   equal(view.$().text(), '');
 });
 
-QUnit.test('should update the block when object passed to #if helper changes', function() {
```

Comment:
Removed without replacement (?). Seems like this is about the content of the bock get updated? I can add it back if it is important.